CHANGELOG.md

```diff
@@ -29,6 +29,7 @@
 ## Breaking changes
 
 - Deprecate functionality to load parameter set from a csv file. Parameter sets must now be provided as python dictionaries ([#2959](https://github.com/pybamm-team/PyBaMM/pull/2959))
+- PyBaMM now has optional dependencies that can be installed with `pip install pybamm[option]` e.g. `pybamm[dev]` ([#3044](https://github.com/pybamm-team/PyBaMM/pull/3044))
```

Comment:
This would have to go to the unreleased section.

Reply:
I think you ended up creating duplicate entries.

Reply:
Yep 😅, fixed that.